### PR TITLE
minor Localization fix

### PR DESCRIPTION
**Pull request by Gedevan-Aleksizde**
* the missing title "Land Management" fixed
* one localizable text added

### CODE REVIEW

File: Agriculture_Estate/AgricultureEstate.csproj [hunk 1]
```diff
@@ -28,6 +28,9 @@
     </ItemGroup>
 
     <ItemGroup>
+      <Reference Include="Bannerlord.BLSE.LauncherEx">
```
Comment (DasNeo) [inline Agriculture_Estate/AgricultureEstate.csproj]: Why is that needed?

Comment (Gedevan-Aleksizde) [reply Agriculture_Estate/AgricultureEstate.csproj]: my bad, it's a mistake. will be fixed later. please wait a few minutes
